### PR TITLE
3Kz1CHwY, abybfzqJ Header

### PR DESCRIPTION
**Pull request by YevgenChop**
- Added Avatar component/stories
- Added Header component/stories
- Added Logo component
- Added Typography components/stories

- Fixed footer bug
- Fixed first-level items are ON at the same time

### CODE REVIEW

**Comment by YevgenChop** [comment]
[Bug navigation](https://trello.com/c/3Kz1CHwY/22-bug-navigation)

**Comment by YevgenChop** [comment]
[Header](https://trello.com/c/abybfzqJ/14-header)

**Comment by giardiv** [comment]
Great job, just the logo seems to be displayed strangely. Just attached the correct SVG


[white.svg.zip](https://github.com/logicalclocks/quartz/files/5333628/white.svg.zip)

Screenshot 2020-10-06 at 14 09 52

File: src/components/avatar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+// Styles
+import styles from './avatar.styles';
+// Default Image
+import defaultImage from './default.svg';
+
+export interface AvatarProps extends Omit<BoxProps, 'css'> {
+  borderColor?: string;
+  src?: string;
+  size?: string;
```
Comment (giardiv) [inline src/components/avatar/index.tsx:13]: For now, the avatar shall not change size. You can just remove this prop

Comment (giardiv) [reply src/components/avatar/index.tsx:13]: The siwe is set at 32px height and width

**Review by giardiv — CHANGES_REQUESTED** [review]

**Comment by giardiv** [comment]
FYI

- the badges avatar will be updated with the dedicated [Trello card](https://trello.com/c/iJjzqRLJ/23-badges)
- great initiative to have documented the font

**Pull request merged**